packages/dds/tree/src/feature-libraries/typed-schema/view.ts

```diff
@@ -144,18 +143,8 @@ export class ViewSchema {
 	}
 
 	private adaptTree(original: TreeNodeStoredSchema): TreeNodeStoredSchema {
-		const objectNodeFields: Map<FieldKey, TreeFieldStoredSchema> = new Map();
-		for (const [key, schema] of original.objectNodeFields) {
-			// TODO: support missing field adapters.
-			objectNodeFields.set(key, this.adaptField(schema));
-		}
-		// Would be nice to use ... here, but some implementations can use properties as well as have extra fields,
-		// so copying the data over manually is better.
-		return {
-			mapFields: original.mapFields,
-			leafValue: original.leafValue,
-			objectNodeFields,
-		};
+		// TODO: support adapters like missing field adapters.
+		return original;
```

Comment:
I don't have the knowledge to understand this change. What happened here?

Reply:
placeholder logic for future adapter based schema evolution no longer compiled, so I no-oped it.